packages/server/envVariables.json

```diff
@@ -0,0 +1,8 @@
+{
+  "POSTGRES_PASSWORD": "Z0hiR5D7BYA43bKJvnMY",
+  "POSTGRES_DB": "postgres",
+  "POSTGRES_USER": "postgres",
+  "POSTGRES_HOST": "croton-global.cizhyv33sdan.us-east-2.rds.amazonaws.com",
+  "POSTGRES_PORT": "5432",
+  "PORT": "3000"
+}
```

Comment:
That's too unsecure for an open DB, let's keep it env vars and pass them as env vars to lambda too.